docs/resources/cluster.md

```diff
@@ -1,100 +1,334 @@
+  }
+
+  instance_type = "aws:c5.large"
+  password      = resource.random_password.password.result
+  pg_config {
+    name  = "application_name"
+    value = "created through terraform"
+  }
+
+  pg_config {
+    name  = "array_nulls"
+    value = "off"
+  }
+
+  storage {
+    volume_type       = "gp3"
+    volume_properties = "gp3"
+    size              = "4 Gi"
+  }
+
+  pg_type               = "epas"
+  pg_version            = "14"
+  private_networking    = false
+  cloud_provider        = "aws"
+  read_only_connections = true
+  region                = "us-east-1"
+}
+
+output "password" {
+  sensitive = true
+  value     = resource.biganimal_cluster.ha_cluster.password
+}
+
+output "ro_connection_uri" {
+  value = resource.biganimal_cluster.ha_cluster.ro_connection_uri
+}
+```
+
+## Extreme High Availability Cluster Example
+```terraform
+terraform {
+  required_providers {
+    biganimal = {
+      source  = "biganimal"
+      version = "0.3.1"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "3.3.1"
+    }
+  }
+}
+
+resource "random_password" "password" {
+  length           = 16
+  special          = true
+  override_special = "!#$%&*()-_=+[]{}<>:?"
+}
+
+variable "cluster_name" {
+  type        = string
+  description = "The name of the cluster."
+}
+
+
+resource "biganimal_cluster" "eha_cluster" {
+  cluster_name = var.cluster_name
+
+  allowed_ip_ranges {
+    cidr_block  = "127.0.0.1/32"
+    description = "localhost"
+  }
+
+  allowed_ip_ranges {
+    cidr_block  = "192.168.0.1/32"
+    description = "description!"
+  }
+
+  backup_retention_period = "6d"
+  cluster_architecture {
+    id    = "eha"
+    nodes = 5
+  }
+
+  instance_type = "aws:c5.large"
+  password      = resource.random_password.password.result
+  pg_config {
+    name  = "application_name"
+    value = "created through terraform"
+  }
+
+  pg_config {
+    name  = "array_nulls"
+    value = "off"
+  }
+
+  storage {
+    volume_type       = "gp3"
+    volume_properties = "gp3"
+    size              = "4 Gi"
+  }
+
+  pg_type            = "epas"
+  pg_version         = "14"
+  private_networking = false
+  cloud_provider     = "aws"
+  region             = "us-east-1"
+}
+
+output "password" {
+  sensitive = true
+  value     = resource.biganimal_cluster.eha_cluster.password
+}
+
+output "connection_uri" {
+  value = resource.biganimal_cluster.eha_cluster.connection_uri
+}
+```
 
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Required
 
-- `cloud_provider` (String) Cloud Provider
-- `cluster_architecture` (Block List, Min: 1) Cluster Architecture (see [below for nested schema](#nestedblock--cluster_architecture))
+- `cloud_provider` (String) Cloud Provider. e.g. "aws" or "azure".
+- `cluster_architecture` (Block List, Min: 1) Cluster Architecture. See https://www.enterprisedb.com/docs/biganimal/latest/overview/02_high_availability/ for details. (see [below for nested schema](#nestedblock--cluster_architecture))
```

Comment:
```suggestion
- `cluster_architecture` (Block List, Min: 1) Cluster Architecture. See [Supported cluster types ](https://www.enterprisedb.com/docs/biganimal/latest/overview/02_high_availability/) for details. See also [schema information](#nestedblock--cluster_architecture).
```

Reply:
I will check the markdown clickable link part (probably it's doable). The `see [below for nested schema]` is coming from the provider generator, which I explained [in this comment](https://github.com/EnterpriseDB/terraform-provider-biganimal/pull/74#discussion_r1033576476).

Reply:
Clickable link is added at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/0aea7392df32d1d3b7fcb6e35524aa585fa8a660